### PR TITLE
Implement task list method

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1800/

File: AppTaskQueue/appscale/taskqueue/task.py [hunk 1]
```diff
@@ -84,17 +99,44 @@ def validate_info(self):
         raise InvalidTaskInfo(
           'Invalid task info: {}={}'.format(attribute, value))
 
+  def get_eta(self):
+    """ Returns the ETA for a task.
+
+    Raises:
+      InvalidTaskInfo if
```
Comment (menivaitsi) [inline AppTaskQueue/appscale/taskqueue/task.py]: Please finish this comment.

File: AppTaskQueue/appscale/taskqueue/task.py [hunk 2]
```diff
@@ -84,17 +99,44 @@ def validate_info(self):
...
+    try:
+      return self.enqueueTimestamp
+    except AttributeError:
+      raise InvalidTaskInfo('No ETA info for {}'.format(self))
+
+  def expired(self, max_retries):
+    """ Checks whether or not a task has expired.
+
+    Args:
+      max_retries: An integers specifying the queue's task retry limit.
```
Comment (menivaitsi) [inline AppTaskQueue/appscale/taskqueue/task.py]: integers -> integer

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1804/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1811/

**Pull request merged**